components/automate-ui/src/app/services/projects-filter/projects-filter.effects.ts

```diff
@@ -42,7 +45,7 @@ export class ProjectsFilterEffects {
 
   private loadOptionsAction$(): () => Observable<ProjectsFilterActions> {
     return () => this.requests.fetchOptions().pipe(
-      map((fetched: AuthorizedProjectsResponse) => {
+      map((fetched: GetProjectsSuccessPayload) => {
```

Comment:
This file contains what is really all the front-end tweaks needed to use names instead of ids.